From 00fd9ab8bf435658616f3f25b18d51b0a7ee5769 Mon Sep 17 00:00:00 2001
From: FX Coudert <fxcoudert@gmail.com>
Date: Sat, 12 Dec 2020 16:18:59 +0100
Subject: [PATCH] Remove legacy poppler-cairo dependency (#69)

poppler-glib already depended on it, so this is non-breaking,
and it is removed in poppler 0.18.1.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 382e220..2e9b6e5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -41,7 +41,7 @@ AC_LANG(C++)
 dnl === Library checks ===
 
 PKG_CHECK_MODULES(POPPLER,
-                  [poppler-cairo >= 0.10 poppler-glib >= 0.10 cairo-pdf])
+                  [poppler-glib >= 0.10 cairo-pdf])
 
 AM_OPTIONS_WXCONFIG
 AM_PATH_WXCONFIG([3.0.0], [wxfound=1], [wxfound=0], [core,base])
